data/sql/updates/pending_db_world/rev_1668358064214502500.sql

```diff
@@ -0,0 +1,900 @@
+(12083, 'esES', '¿Están usándolo para dar más poder a su ejército? No tiene sentido, los cazadores de magos no son tan poderosos.$b$bEsto tiene que significar que solo usan la magia en el punto final del lugar hacia el que están canalizando la línea Ley.$b$b¡Vamos a necesitar más información si queremos ser capaces de detenerlos!', 0),
+(12084, 'esES', '¿Están usándolo para dar más poder a su ejército? No tiene sentido, los cazadores de magos no son tan poderosos.$b$bEsto tiene que significar que solo usan la magia en el punto final del lugar al que están canalizando la línea Ley.$b$b¡Vamos a necesitar más información si tengo que idear un plan para detenerlos!', 0),
+(12085, 'esES', 'Jajajajajaja... uf uf, qué conmovedor.$b$b<La condescendencia en la profunda risa del señor supremo es evidente.>$b$b¿Qué dije? ¡Traidores! Aún así, conocí a Deino en una ocasión. ¡Ella al menos es una troll con honor!$b$bMe encargaré de que averigüe lo que le sucedió a su hermano. Incluso olvidaré decirle que fuiste tú quien lo mató.$b$b<Agmar se fija en ti con una mirada calculadora.>$b$b¡Ahora te tengo, $c!', 0),
+(12089, 'esES', 'No es el primero de esos hechiceros cobardes en traicionarnos, y seguro que tampoco será el último. Pero qué pena también... Era uno de nuestros mejores magos. Podía invocar un buffet que te hacía la boca agua y a tu corazón dar brincos de alegría...$b$b¡BUÁ!', 0),
+(12090, 'esES', 'Tienes a ese feo hijo de un caballo, ¿eh? Bueno, eso es una molestia menos con la que nuestros soldados tendrán que lidiar en estas tierras congeladas.', 0),
+(12091, 'esES', '¡Ya era hora de que alguien pusiera fin a la matanza de esa bestia asquerosa! Muy buen trabajo, $N.', 0),
+(12092, 'esES', '¡Gracias, $r! Hemos perdido tanto ya. Gracias a ti no perderemos a los más majestuosos de los niños de Rasganorte.$b$bPronto partiré para reunirme con los ancestros que salvaste para ver si nos pueden ayudar en estos momentos de apuros.', 0),
+(12095, 'esES', '¿Qué te hace pensar que has tenido éxito donde Rokhan falló?', 0),
+(12096, 'esES', 'Te doy las gracias, $c.$b$bEn cuanto termine de observar lo que esos asquerosos boticarios están tramando, me dirigiré al bosque para ver si puedo convencer a esos ancestros a que se unan a nosotros.$b$bSi no, ¡los convenceré de que no ataquen a quienes los salvaron!', 0),
+(12097, 'esES', '¡JA! ¡Lo has hecho! Los soldados en la Puerta de Cólera están en deuda contigo, $N! Esto seguramente convenza a alguno de ellos para estar otro día más en la tierra.', 0), -- This'll surely buy a few of them another day above ground. <-- No tengo ni papa de lo que pueda significar esta mierda, se que la traducción literal no tiene sentido, tiene que ser algo concreto.
```

Comment:
Remove the profanations, please. No aceptamos comentarios, tanto en cpp como en db, que contengan vocabulario vulgar.

Reply:
> Remove the profanations, please. No aceptamos comentarios, tanto en cpp como en db, que contengan vocabulario vulgar. Other than that, all good.

Hostias se me olvidó quitarlo.
Thx i forgot to remove it 🥵🥵🥵🥵🥵

Reply:
Np, thank you!